anagram/tests/anagram.rs

```diff
@@ -1,11 +1,6 @@
-#![crate_name = "anagram_test"]
-#![crate_type = "lib"]
-#![feature(collections)]
-
-mod anagram;
+extern crate anagram;
 
 #[test]
-#[ignore]
```

Comment:
Yeah, I do wish table-driven tests were more common in other language communities. You're right though. I don't really see it happening in JavaScript or Ruby. And since exercism does seem to be a little bit about getting a feel for the rhythm and process of TDD it seems like it would be wrong to introduce table-driven tests in a language track where nobody does that out in the wild.